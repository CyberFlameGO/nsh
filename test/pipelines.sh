echo foo; echo bar
echo hello | echo world | hexdump -C
THIS_IS_ENV_VAR=abc echo | THIS_IS_ENV_VAR=123 env | grep THIS_IS_ENV_VAR
