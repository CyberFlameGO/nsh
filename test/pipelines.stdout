foo
bar
00000000  77 6f 72 6c 64 0a                                 |world.|
00000006
THIS_IS_ENV_VAR=123
